test_candidate.py

```diff
@@ -90,5 +123,39 @@ def test_content(self):
     self.assertTrue(match)
 
 
+  def test_wikipedia_html(self):
+    """Test parsing wikipedia html.
```

Comment:
Maybe add something to the comment that mentions the test_house.html? That way someone just reading through the code's method statements will know that there is a possible expansion of this function, if they wanted to re-use functionality. (It's probably not necessary to expand it now, but it seems like a useful note to leave in)